src/components/molecules/Tab/TabArea/TabArea.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+
+import type { ReactNode } from "react";
+
+import { tabContext } from "../context";
+
+export const TabArea = ({
+  children,
+  defaultPage = 1,
+}: {
+  children: ReactNode;
+  defaultPage?: number;
+}) => {
+  const [currentPage, setCurrentPage] = useState(defaultPage);
+
+  return (
+    <tabContext.Provider
+      value={{
+        currentPage,
+        setCurrentPage,
+      }}
+    >
+      {children}
+    </tabContext.Provider>
```

Comment:
Could the context also start with a capital letter as React components usually do?
```suggestion
import { TabContext } from "../context";

export const TabArea = ({
  children,
  defaultPage = 1,
}: {
  children: ReactNode;
  defaultPage?: number;
}) => {
  const [currentPage, setCurrentPage] = useState(defaultPage);

  return (
    <TabContext.Provider
      value={{
        currentPage,
        setCurrentPage,
      }}
    >
      {children}
    </TabContext.Provider>
```